### DISCUSSION TITLE
Unmyelinated axons

### DISCUSSION THREAD

**Discussion by shinysyk** [neuropoly/axondeepseg#692] [Help needed]
Hi. Thanks to your help, I was able to successfully install axondeepseg. Then, I played with your example dataset at https://axondeepseg.readthedocs.io/en/latest/documentation.html#manual-correction-of-segmentation-masks. One thing that immediately caught my attention is that axondeepseg cannot detect unmyelinated axons. This is a very important aspect from the biological perspective. Typically, one is interested in 1) how many axons are there, 2) what fraction of them are myelinated (or unmyelinated), 3) myelin thickness, and 4) axon size (radius). Would it be possible to detect all axons regardless of myelin?

**Comment by mathieuboudreau** [comment]
Hi @shinysyk,

In order for deep learning models to learn how to segment different objects, ground truth (typically manually segmented images performed by humans) must be provided; do you have a set of histological images that you've manually segmented the unmyelinated axons for?

**Reply by shinysyk** [reply]
No, I do not have. How many images do you need for good training? We may start with some of the images that you used for original training. I may be able to annotate them. At the moment, I am interested in TEM data. If you want, we can communicate further via email.
